### PR TITLE
Rf2 eei eligibility all tests passing

### PR DESCRIPTION
**Pull request by k8mckay**
RF2 ESC and PRCs EEI for eligibility

### CODE REVIEW

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_calculation.py [hunk 1]
```diff
@@ -136,14 +136,31 @@ class RF2_ESC_calculation(Variable):
...
+      replacement_activity = buildings('RF2_replacement_activity', period)
+      EEI_eligible_replacement = buildings('RF2_product_EEI_ESC_replacement_eligibility', period)
+      EEI_eligible_install = buildings('RF2_product_EEI_ESC_install_eligibility', period)
+
+      RF2_eligible_ESCs = np.select(
+            [
+                replacement_activity * EEI_eligible_replacement,
+                np.logical_not(replacement_activity) * np.logical_not(EEI_eligible_replacement),
+                replacement_activity * EEI_eligible_install,
+                np.logical_not(replacement_activity) * EEI_eligible_install
```
Comment (alisha17) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_calculation.py:150]: we should also add 

false replacement_activity and false EEI_eligible_install

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_variables.py [hunk 2]
```diff
@@ -36,67 +36,135 @@ class RF2_total_display_area(Variable):
...
+    definition_period = ETERNITY
+
+    def formula(building, period, parameters):
+      product_EEI = building('RF2_product_EEI', period)
+      product_class_5 = building('RF2_product_class', period)
+        
+      product_EEI_to_check_ESC = np.select(
+          [
+              (product_EEI < 51),
+              (product_EEI >= 51) * (product_EEI < 81) * (product_class_5 != RF2ProductClass.product_class_five),
```
Comment (alisha17) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_variables.py:110]: this should be

np.logical_not(product_class_5 == RF2ProductClass.product_class_five)

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_variables.py [hunk 3]
```diff
@@ -36,67 +36,135 @@ class RF2_total_display_area(Variable):
...
+    definition_period = ETERNITY
+
+    def formula(building, period, parameters):
+        product_EEI = building('RF2_product_EEI', period)
+        product_class_5 = building('RF2_product_class', period)
+        
+        product_EEI_to_check_PRC = np.select(
+            [
+              (product_EEI < 51),
+              (product_EEI >= 51) * (product_EEI < 81) * (product_class_5 != RF2ProductClass.product_class_five),
```
Comment (alisha17) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_variables.py:155]: replace with np.logical_not

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_PRC_calculation.py [hunk 4]
```diff
@@ -96,14 +96,21 @@ def formula(buildings, period, parameters):
...
+
 
         RF2_eligible_PRCs = np.select(
             [
-                replacement_activity,
-                np.logical_not(replacement_activity)
+                replacement_activity * EEI_eligible_replacement ,
+                np.logical_not(replacement_activity) * np.logical_not(EEI_eligible_replacement),
+                replacement_activity * EEI_eligible_install,
+                np.logical_not(replacement_activity) * EEI_eligible_install
```
Comment (alisha17) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_PRC_calculation.py:108]: I think we should add all combinations for the three variables

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_calculation.py (hunk 1 above)
Comment (alisha17) [reply openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_calculation.py:150]: I think we should add all combinations for the three variables

Comment (k8mckay) [reply openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/RF2/certificate_estimation/RF2_ESC_calculation.py:150]: Adding these extra conditions is breaking the tests, so we are not going to do this.

**Review by alisha17 — APPROVED** [review]

**Pull request merged**